cfpb/curriculum-review-tool
# 3 of 3: Crtool storage changes

Allows Curriculum review sessions to be saved on the server alongside localStorage.
This allows a person to continue working on a Curriculum Review from another device by using the same unique URL that contains an access code.

Note: This PR must be merged after PR: #404 
---

## Additions

- a localStorage wrapper class that handles storing values in localStorage and syncing those values to the server.
- additional unit tests for new localStorage wrapper class


## Changes

- Replace embedded JavaScript from the crt-start.html with dedicated analytics.js and startPage.js files.


## How to test this PR

1. Go to: /practitioner-resources/youth-financial-education/curriculum-review/before-you-begin/
2. Start a new Curriculum Review
3. You'll notice a token string is added to the end of the URL


## Screenshots


## Notes and todos

-


## Checklist




### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
